recipe/conda_build_config.yaml

```diff
@@ -329,6 +329,8 @@ pin_run_as_build:
     max_pin: x.x
   tk:
     max_pin: x.x
+  tiledb:
+    max_pin: x.x.x
```

Comment:
Can this be moved to `run_exports` of `tiledb`?

Reply:
I'll do that in a new release but right now I want to use the old `1.6.0` and I don't think that has `run_exports`. (Also, I'm preparing for a workshop with limited time to fix things, so I'm going for the shortest path here.)